django/core/files/storage/filesystem.py

```diff
@@ -192,14 +191,15 @@ def delete(self, name):
             # concurrently.
             pass
 
-    def exists(self, name):
-        try:
-            exists = os.path.lexists(self.path(name))
-        except SuspiciousFileOperation:
-            raise
+    def get_available_name(self, name, max_length=None):
+        # Ensure protection against insecure names.
```

Comment:
This shouldn't be necessary anymore with the CVE fix, right?

Reply:
You're right that with the CVE the same validation is applied elsewhere and so it is not strictly necessary